arbnode/batch_poster.go

```diff
@@ -1241,7 +1253,7 @@ func (b *BatchPoster) maybePostSequencerBatch(ctx context.Context) (bool, error)
 		}
 
 		var useEigenDA bool
-		if b.eigenDAWriter != nil {
+		if b.eigenDAWriter != nil && !b.failoverETHDA {
```

Comment:
add comment explaining the logic here. Why are you setting useEigenDA to true only when we don't failover?

Reply:
Don't we still want to use eigenda until we failover?